public/js/specberus.js

```diff
@@ -241,8 +241,8 @@ jQuery.extend({
             ,   patentPolicy = $patentPolicy.find('label.active').attr('id')
             ;
             profile = $profile.val();
-            if (!url) addMessage(MSG_ERROR, 'Missing “URL” parameer');
-            if (!profile) addMessage(MSG_ERROR, 'Missing “profile” parameter');
+            if (!url) addMessage(MSG_ERROR, {message: 'Missing “URL” parameer'});
```

Comment:
[Fixed](https://github.com/w3c/specberus/pull/506/commits/9ad0d34c7dfd2e8aa99c39cea7866760d0430d33), thank you!